interface/patient_file/summary/pnotes_full.php

```diff
@@ -223,11 +223,14 @@
     });
 
     // load divs
-    $("#stats_div").load("stats.php",
+
+    // I can't find a reason to load this!
+    /*$("#stats_div").load("stats.php",
         {
             csrf_token_form: <?php echo js_escape(CsrfUtils::collectCsrfToken()); ?>
         }
-    );
+    );*/
```

Comment:
That does seem odd. Would just remove it.

Reply:
lol, befuddled me too! I hate coming across stuff like this because, I end up pondering and fretting with; Well it's there. Must be a reason! :)